examples/git.py

```diff
@@ -1,9 +1,31 @@
-from pyinfra.modules import git
+from pyinfra import host
+from pyinfra.modules import apk, apt, files, git
 
 SUDO = True
 
+linux_name = host.fact.linux_distribution.get('name', '')
```

Comment:
This can just be `host.fact.linux_name`.

Reply:
Ok. Thanks.